extensions/wikia/EmbeddableDiscussions/scripts/ext.embeddableDiscussions.js

```diff
@@ -10,14 +15,43 @@ require([
 		trackingMethod: 'analytics'
 	});
 
-	$(function() {
+	function openModal(link, title) {
+		// Track impression
+		track({
+			action: tracker.ACTIONS.IMPRESSION,
+			label: 'embeddable-discussions-share-modal-loaded',
+		});
+
+		uiFactory.init(['modal']).then(function (uiModal) {
+			var modalConfig = {
+				vars: {
+					classes: ['embeddable-discussions-share-modal'],
+					content: '',
+					id: 'EmbeddableDiscussionsShareModal',
+					size: 'small',
+				}
+			};
+
+			uiModal.createComponent(modalConfig, function (modal) {
+				modal.$content
+					.html(mustache.render(templates.ShareModal, {
+						heading: $.msg('embeddable-discussions-share-heading'),
+						icons: sharing.getData(mw.config.get('wgUserLanguage'), link, title),
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Remove this trailing comma. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3ATrailingComma)